node/bft-consensus/tests/common/state.rs

```diff
@@ -38,24 +40,29 @@ pub type Amount = u64;
 
 // A simple state for BFT consensus tests.
 pub struct TestBftExecutionState {
+    pub committee: SharedCommittee,
     pub balances: Mutex<HashMap<Address, Amount>>,
-    pub processed_txs: AtomicUsize,
+    pub processed_tx_count: AtomicUsize,
+    pub processed_txs: Mutex<HashSet<u64>>,
```

Comment:
do we still need the count if we store the tx ids now?

Reply:
not really